src/main/java/com/parth/jwt/enumPackage/Role.java

```diff
@@ -0,0 +1,7 @@
+package com.parth.jwt.enumPackage;
+
+public enum Role {
```

Comment:
Everyone should use Enum to define the role.

Reply:
Yes